protocol-definitions/MC.yaml

```diff
@@ -643,6 +643,14 @@ methods:
             0 - DISCARD_AFTER_MUTATION
             1 - THROW_EXCEPTION
             2 - THROW_EXCEPTION_ONLY_IF_REPLICATION_ACTIVE
+        - name: consistencyCheckStrategy
+          type: byte
+          nullable: false
```

Comment:
@ihsandemir  Can't remember the exact rules here, but wouldn't it be better to mark this as `nullable`?

Reply:
we do not mark primitives as nullable.